src/amo/components/StaticPages/About.js

```diff
@@ -0,0 +1,180 @@
+          </p>
+        </section>
+        <section>
+          <h2>{i18n.gettext('Who creates these add-ons?')}</h2>
+          <p>
+            {i18n.gettext(`The add-ons listed here have been created by
+              thousands of developers from our community, ranging from
+              individual hobbyists to large corporations. All publicly
+              listed add-ons are reviewed by a team of editors before
+              being released. Add-ons marked as Experimental have not
+              been reviewed and should only be installed with caution.`)}
+          </p>
+        </section>
+        <section>
+          <h2>{i18n.gettext(`How do I keep up with what's happening at AMO?`)}</h2>
+          <p>
+            {i18n.gettext(`There are several ways to find out the latest news from
+              the world of add-ons:`)}
+          </p>
+          <ul>
+            <li
+              // eslint-disable-next-line react/no-danger
+              dangerouslySetInnerHTML={
+                sanitizeHTML(
+                  i18n.sprintf(
+                    i18n.gettext(`Our %(startLink)sAdd-ons Blog%(endLink)s
+                      is regularly updated with information for both add-on
+                      enthusiasts and developers.`),
+                    { startLink: '<a href="https://blog.mozilla.com/addons/">', endLink: '</a>' }
+                  ), ['a'])
+              }
+            />
+            <li
+              // eslint-disable-next-line react/no-danger
+              dangerouslySetInnerHTML={
+                sanitizeHTML(
+                  i18n.sprintf(
+                    i18n.gettext(`We often post news, tips, and tricks to our Twitter account,
+                      %(startLink)smozamo%(endLink)s`),
+                    { startLink: '<a href="https://twitter.com/mozamo">', endLink: '</a>' }
+                  ), ['a'])
+              }
+            />
+            <li
+              // eslint-disable-next-line react/no-danger
+              dangerouslySetInnerHTML={
+                sanitizeHTML(
+                  i18n.sprintf(
+                    i18n.gettext(`Our %(startLink)sforums%(endLink)s are a good place to
+                      interact with the add-ons community and discuss upcoming changes to AMO.`),
+                    { startLink: '<a href="https://discourse.mozilla-community.org/c/add-ons">', endLink: '</a>' }
+                  ), ['a'])
+              }
+            />
+          </ul>
+        </section>
+        <section>
+          <h2>{i18n.gettext('This sounds great! How can I get involved?')}</h2>
+          <p>{i18n.gettext(`There are plenty of ways to get involved. If you're on the technical side:`)}</p>
+          <ul>
+            <li>
+              <Link to={'/developers'}>{i18n.gettext('Make your own add-on.')}</Link>
+              <span> {i18n.gettext('We provide free hosting and update services and can help you reach a large audience of users.')}</span>
```

Comment:
That's deliberate. Because we don't have a Link equivalent of JS otherwise the leading link would be substituted in. I think this would be nice to fix, but I didn't want to get into it. Can definitely file an issue for that one. I meant to call it out in the comments.